en/views.rst

```diff
@@ -460,7 +462,7 @@ Contacts plugin::
     {
         public function view_active()
         {
-            $this->layout = 'Contacts.contact';
+            $this->getView()->layout('Contacts.contact');
         }
```

Comment:
I wonder if that is going to be confusing for people. That was one nice part to the controller properties. The order in which you did things didn't matter much. As long as everything was done before the view was created.

I can see this potentially going funny if someone is using RequestHandler and they set the `layout`. Previously things would just work, but now an HTML view will be rendered.